baggers.js

```diff
@@ -14081,6 +14082,55 @@ var data = {
         },
       ],
     },
+    {
+      since: '2018-03-06',
+      name: 'Claire Zuliani',
+      bio:
+        "Outilleuse de projets à la croisée du numérique et des humains. Venue du monde associatif et culturel et reconvertie dans le développement web, je conseille aujourd'hui les structures engagées et développe des outils (surtout en ruby et javascript). J'accompagne aussi sur un plan humain en amont et durant les projets : facilitation, intelligence collective, user experience...",
+      picture:
+        'https://d33wubrfki0l68.cloudfront.net/dddd2605dce105601dcba9587f14633175792c58/82039/assets/images/avatar.png',
+      location: 'Epinal',
+      cities: ['Epinal', 'Nancy', 'Metz'],
+      contacts: {
+        twitter: 'ZooletteDesBois',
+        mail: 'clairezuliani@gmail.com',
+      },
+      websites: [
+        {
+          name: 'Portfolio',
+          url: 'https://www.clairezuliani.com',
+        },
+        {
+          name: 'Carnet',
```

Comment:
Not supported name. See the list in the README.

Reply:
oh, sorry ! I hadn't noticed the restricted list !